<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontWidget</class>
 <widget class="QWidget" name="FontWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Family:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Family font.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Style:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="fontStyleBoldButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Bold</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/icons/resources/icons/app-icons/bold.png</normaloff>:/icons/resources/icons/app-icons/bold.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fontStyleItalicButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Italic</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="rsrc.qrc">
           <normaloff>:/icons/resources/icons/app-icons/italic.png</normaloff>:/icons/resources/icons/app-icons/italic.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="fontSizeSpinner">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Font point size.</string>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="toolTip">
        <string>Available chars for text entry.</string>
       </property>
       <property name="text">
        <string>Chars left:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLCDNumber" name="charCounter">
       <property name="toolTip">
        <string>Available chars for text entry.</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="numDigits">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>512.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="fontTextArea">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
